auslib/util/sentry.py

```diff
@@ -0,0 +1,13 @@
+from raven.processors import Processor
+
+
+class SanitizeHeadersProcessor(Processor):
```

Comment:
It looks like this is the default now, based on https://github.com/getsentry/raven-python/blob/80490488fb5c62d927ac789313f971e02ee395e2/raven/conf/defaults.py - let's get rid of this!